src/Compare.js

```diff
@@ -69,69 +76,56 @@ class Compare extends Component {
   render() {
     return (
       <div className='compare-page'>
```

Comment:
Thank you for walking me through this section, I am clear on what it is rendering and it looks really clean.

No edits.